recipes/basemap-data-hires/meta.yaml

```diff
@@ -0,0 +1,29 @@
+package:
+    name: basemap-data-hires
```

Comment:
That was once discussed on the conda mailinglist, but the conversation went dead (I proposed something a little outlandish as I recall).